.github/workflows/playwright-playground.yml

```diff
@@ -0,0 +1,42 @@
+name: Playwright Tests (Playground)
+on:
+  push:
+    branches: [ playground ]
+  pull_request:
+    branches: [ playground ]
```

Comment:
Unsure if this is true. People will have code changes that won't be deployed on the target URLs. Right? Am I missing something?